manualTests/build.gradle.kts

```diff
@@ -173,29 +253,105 @@ xtcRun {
      * Currently, all module configurations in the xtcRun DSL will be executed in sequence,
      * as their order declared.
      *
-     * We suspect that the pre-generated run tasks from the XTC Plugin are not the most optimal
-     * and intuitive way of running XTC modules. It's probably cleaner for the user to modify
-     * the configuration on task level for any runtime task, and add many simple custom run tasks
-     * for clarity. However, we haven't ahd the cycles to support the standard overrides for
-     * modules to run (all other DSL can be overridden) at task level. It is easy and encouraged
-     * to contribute to the XTC Plugin build DSL, so that we get more functionality, clearer
-     * and shorter syntax, and new features that we feel we require. It's not clear what all
-     * of these are, but working with an XTC project that applies the XTC Plugin will likely
-     * discover most of the shortcomings quickly, so that we can file enhancement requests.
-     *
-     * TODO: Add parallelism, and a simpler way to work with this.
-     * TODO: Add a nicer DSL syntax with a nested modules section.
+     * The project-global run configuration is mostly modelled on the Java Application plugin,
+     * which has a single configuration with mainClass, and a single run task. This makes sense,
+     * because if you added individual run tasks for individual modules, they would have to
+     * accept inputs (from where?), which we sort of do when it comes to the simple test
+     * default, but they would also not be a standard "here is my outputs" Gradle task. For
+     * a new task, registered without any outputs in the configuration, Gradle will always treat
+     * it as stale (does the equivalent of our Tasks.considerNeverUpToDate()), so out of the box
+     * a run task (or any Gradle task) will not be cached. Of course XTC run tasks can do anything
+     * as well, and they don't declare any outputs for the same reason.
      */
+    val moduleTestName = getTestModuleName() // module name on command line or 'TestSimple' as default
+    // Special case - use the Runner to run all tests in parallel, as long as the xtcRunAll tasks does not
+    // automatically do this with everything in the module path of the source set. Also modify that task
+    // after bugfixing to have a parallel option, through the XTC runner, or whatever it may be.
     module {
-        moduleName = "TestFizzBuzz" // Will add other ways to resolve modules too.
-        showVersion = true // Overrides env showVersion flag.
-        moduleArgs("Hello, ", "World!")
+        if (!shouldRunAllTestsInParallel()) {
```

Comment:
I'd suggest to avoid a double-negative:

if (shouldRunAllTestsInParallel()) {
   ...
} else {
    moduleName = moduleTestName
}